server/games/game.py

```diff
@@ -463,14 +471,21 @@ async def resolve_game_results(self) -> EndedGameInfo:
         await self._run_pre_rate_validity_checks()
 
         basic_info = self.get_basic_info()
+
+        team_army_results = [
+            [self.get_army_results(player) for player in team]
+            for team in basic_info.teams
+        ]
+
         team_outcomes = [GameOutcome.UNKNOWN for _ in basic_info.teams]
 
-        if self.validity is ValidityState.VALID:
+        if self.validity == ValidityState.VALID:
```

Comment:
So looks like you are gonna keep this in for now?

Reply:
Yeah I thought this might be better as a separate PR as it looked like I might have to alter the logic of `game_results.resolve_game` as well and we might want to be able to roll that back easily without undoing the changes in this PR?